metadata/utilities/transmission_lines.md

```diff
@@ -0,0 +1,74 @@
+# Title
+
+Utah Transmission Lines
+
+## ID
+
+e37e9dd4-5c8a-49db-bee9-8006657b2133
+
+## Brief Summary
+
+Polyline dataset of electrical transmission lines in Utah.
+
+## Summary
+
+This dataset represents electrical transmission lines in Utah as digital polylines. Further utilities data can be found in the [Utilities Data Index](https://gis.utah.gov/products/sgid/utilities/) on the SGID.
+
+## Description
+
+### What is the dataset?
+
+This dataset displays transmission lines in Utah as digital polylines. Features in this dataset include standard and high voltages lines as well as sub-transmission lines. This dataset does not contain information on utility companies, pricing, availability, or ownership of transmission lines.
+
+### What is the purpose of the dataset?
+
+This dataset has been made available to the public for general reference and cartographic purposes.
+
+### What does the dataset represent?
+
+Each polyline in this dataset represents the path of an electrical transmission line in Utah. Features in this dataset are labelled by their line type as indicated by the "layer" field.
+
+KV = Voltage of the transmission line in kilovolts.
+SUB-PP = substation power plant?
+SUB-CO =
+
+<!--- Do we know what these abbreviations stand for in the attribute table? --->
```

Comment:
were we wanting to merge this with open questions?

Reply:
I'm still gathering information on this one, so it's not ready to be merged quite yet.

Reply:
sorry i could have sworn this was already merged. my bad.